3-enrich/emr-etl-runner/config/config.yml.sample

```diff
@@ -69,6 +69,7 @@ storage:
     rdb_loader: 0.14.0
     rdb_shredder: 0.13.0        # Version of the Spark Shredding process
     hadoop_elasticsearch: 0.1.0 # Version of the Hadoop to Elasticsearch copying process
+  loader_consistency_check: true # Enable or disable RDB loader S3 consistency checks, defaults to true
```

Comment:
Hm. I thought this should be matter of `snowplow-emr-etl-runner --skip consistency_check`. Cannot we right skip/include `vacuum` and `analyze` this way?

Reply:
mmh true, I'll modify

Reply:
Someday this should be even part of storage target JSON configuration. But right now I believe CLI-options are the way to go.